rust/private/rust.bzl

```diff
@@ -620,7 +620,14 @@ _common_attrs = {
         """),
     ),
     "rustc_flags": attr.string_list(
-        doc = "List of compiler flags passed to `rustc`.",
+        doc = dedent("""\
```

Comment:
hmmm, does this render cleanly? Attribute `doc` fields are typically one continuous line of text and expected to be short...

This is typically why you see the less elegant
```python
doc = (
    "List of compiler flags passed to `rustc`. " + 
    "These strings are subject to Make variable expansion for predefined " +
    "source/output path variables like `$location`, `$execpath`, and `$rootpath`. " +
    "This expansion is useful if you wish to pass a generated file of " +
    "arguments to rustc: `@$(location //:package:target)`."
),
```

Reply:
I just copied the format of all of the other ones (?)

Reply:
it looks like it renders just fine 😄. Disregard!